neco-exporter/pkg/exporter/exporter.go

```diff
@@ -102,26 +117,37 @@ func (e *Exporter) Run(ctx context.Context) error {
 
 	for {
 		var wg sync.WaitGroup
+		var isLeader bool
+		if mgr := manager.PeekManager(); mgr != nil {
+			select {
+			case <-mgr.Elected():
+				isLeader = true
+			default:
```

Comment:
Don't we need to block following codes, until `mgr.Elected()` sends some values?
I wonder that `select` passes through `default` when it cannot receive any values from a channel.

Reply:
There are two possible way to expose metrics from a non-leader instance.
1. Some metrics are allowed to be exposed from non-leader.
2. All metrics should be exposed from leader-only.

Currently this PR implements the first.
- For this (first) choice, it's ok to proceed with `isLeader = false`, it will expose non-leader metrics.
- For the second choice, it should anyway go through with `default:` to expose `neco_cluster_collector_leader 0` indicating it is a non-leader instance. However the implementation should be slightly modified.